playground/frontend/README.md

```diff
@@ -21,8 +21,14 @@
 
 ## About
 
-Apache Beam is an open-source, unified model for defining parallel processing pipelines for batch and streaming data.
-It provides a portable API layer for building sophisticated data-parallel processing pipelines that may be executed across a diversity of execution engines, or runners.
+Apache Beam is an open-source, unified model for defining parallel processing pipelines for batch
+and streaming data. It provides a portable API layer for building sophisticated data-parallel
+processing pipelines that may be executed across a diversity of execution engines, or runners.
+
+## Playground Assets Licence
```

Comment:
If assets are licensed under Apache license is there a reason mention it here? 

As far as I know, all contributions to the Beam repo are considered given to Apache and licensed under the same license as any other part.

Reply:
Hi @aaltay! The thing is that these assets are created by our designer in Akvelon, that is why we wanted to explicitly mention that they are given under Apache license

Reply:
OK. Let's ask someone who would know better than me.

@kennknowles - What do you think about this? Is it needed to mention who created assets before they got into the beam repo?

Reply:
I don't think this sort of statement is necessary. All contributors sign the Apache ICLA (and companies sign the CCLA) - which grants a license to the ASF for all of their contributions.

I don't have an objection to leaving the statement as it is - but I don't think it's necessary. I'm okay with merging as-is or removing.

Reply:
Great, let's then close this PR, thank you @pabloem @aaltay